views/blog.py

```diff
@@ -36,7 +38,10 @@ def get(self, request, format=None):
         sanitized_response = []
         blog_list = json.loads(response.content).get("items")
         max_blog_count = 6
-        required_blog_list = blog_list[:max_blog_count]
+        if response.status_code == 200:
+            required_blog_list = blog_list[:max_blog_count]
+        else:
+            return Response(status=status.HTTP_204_NO_CONTENT)
```

Comment:
Add a message too. "No blog posts exist"

Reply:
i tried to add message in the third commit of this pr. but when ever i add some data or object with status 204 then the request doesnot get passed.
i read about it and its only with the case of status 204 as you cannot send any data or(content) with status 204 .
otherwise it wouldn't be ""NO_CONTENT""

Reply:
Oh, okay. Makes sense!